external/emulation/README.md

```diff
@@ -54,3 +54,21 @@ The microphysics emulation loads a keras model specified by the `TF_MODEL_PATH`
 The model input/output names are used to update the state, so they should match the variables pushed into global state by call_py_fort.  See `GFS_physics_driver.F90` to see the list of available state variables.  Any state field that intersects with a field produced by the emulator will be adjusted to {state_name}_physics_diag so that piggy-backed information is present.
 
 Running with `save_zc_microphysics = True` will save the emulator outputs and diagnostic physics info to file.
+
+
+## Loading TFRecord data
+
+TFRecord is a binary file format that it native to tensorflow. This means that
+loading it requires no pure-python code and should be portable and performant.
+If `SAVE_TFRECORD` is set, then the model will save each rank to a separate
+`.tfrecord` file. Because tfrecords contain raw serialized data, we also save a
+tf module to `parser.tf` to parse it. To open the data, use the
+following boilerplate:
+
+```python
+url = "gs://vcm-ml-experiments/microphysics-emulation/2021-12-13/pzp6alfw/artifacts/20160611.000000/tfrecords"
+parser = tf.saved_model.load(f"{url}/parser.tf")
+tf_ds = tf.data.TFRecordDataset(
+    tf.data.Dataset.list_files(f"{url}/*.tfrecord")
+).map(parser.parse_single_example)
```

Comment:
Is it possible to load remote data with fsspec?

Should we have a helper function in `vcm` or in `loaders` to simplify this? Possibly one that enables remote data through a local tempdir. I would still keep this explicit tensorflow example if you add a helper.

Reply:
tf supports remote gs links natively. This code actually loads the data at that URL. There are performance implications to this...which is why i exposed the raw parsing funtions.

Reply:
I'll say it's OK if you want to merge without one, but a helper function would make it a lot easier for me to work with this data format. This is a lot of minor name and operation details to remember. I feel the same way every time I need to open a zarr (it's always a google search), and that's a simpler set of operations. I don't currently need to work with this data format, so it's not important for now.

Reply:
Sure. we can add a higher level api to `fv3fit` in the future...would be nice for notebook use. Only reason i didn't is that this "higher level" API would tend to grow many options...e.g. shuffle the file names before loading...number of processors for the `.map`...batch size...etc. At some point it's easier to just have the 4 lines of boiler plate rather than a helper with 10 arguments.